src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="img"
+        class="card__photo"
+      >
+      <div class="card__name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="card__code">
+        Код товара: 195434
+      </div>
+      <div class="card__rating">
+        <div class="card__stars stars_4">
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+        </div>
+        <div class="card__feedbacks">
+          Отзывов: 5
+        </div>
+      </div>
+      <div class="card__price">
+        <div class="card__prop">
+          Цена:
+        </div>
+        <div class="card__value">
+          69 999 грн
+        </div>
+      </div>
+      <a
+        href="#"
+        class="card__btn"
+        data-qa="hover"
+      >
+        КУПИТЬ
+      </a>
+    </div>
```

Comment:
will be great to add some empty lines. It improves code readability

Reply:
added empty lines(maybe to many, but check it pls and give feedback)